build/templates/_grpc.py/numpy_read_method.py.mako

```diff
@@ -0,0 +1,12 @@
+<%page args="f, config, method_template"/>\
```

Comment:
I didn't create a separate "grpc_filename" field in the method_templates dict in the .hapigen metadata. Instead, I'm piggy-backing off of the library_interpreter_filename, because the customizations required for grpc are almost entirely 1-to-1 (except for the new lock/unlock customizations I just added for grpc only, for which the library_interpreter variant just includes the default_method template. So my filenames need to match the library_interpreter filenames.